content/polyhedra/components/polyhedron.less

```diff
@@ -0,0 +1,21 @@
+// =============================================================================
+// Polyhedron Component
+// (c) Mathigon
+// =============================================================================
+
+
+@import "../../shared/variables";
+
+x-polyhedron {
+  display: block;
+  position: relative;
+  margin: 0 auto;
+  canvas { max-width: 100%; height: auto; }
+
+  .label3d {
+    position: absolute;
+    line-height: 1.2;
+    font-style: italic;
+    pointer-events: none;
+  }
+}
```

Comment:
Don't duplicate this file. We can just do `x-solid, x-polyhedron` in the existing CSS file.

Reply:
What about replacing the duplicate styles with: 
```
@import "../../shared/components/solid.less";

x-polyhedron {
  &:extend(x-solid all);
}
```
; That way usage of the polyhedron component still involves importing the x-polyhedron less file rather than the x-solid less file.